pkg/apiserver/experiment/experiment.go

```diff
@@ -86,6 +86,7 @@ func Register(r *gin.RouterGroup, s *Service) {
 	endpoint.POST("/new", s.createExperiment)
 	endpoint.GET("/detail/:kind/:namespace/:name", s.getExperimentDetail)
 	endpoint.DELETE("/:kind/:namespace/:name", s.deleteExperiment)
+	endpoint.PUT("/update", s.updateExperiment)
```

Comment:
I am just wondering do we need to make `kind`, `namespace` and `name` in the API endpoint? We don't do this for `/new` as well. What do you think? @cwen0

Reply:
These messages are included in the request body, I think it is unnecessary to add them in endpoint and they are repeated messages.

Reply:
I mean remove name and namespace from body and add them to URL. Do you think this makes sense?

Reply:
I think it is ok at present, if we need this way, we can update this in the future.

Reply:
Actually I am not against the current way, but let's choose one solution and don't change it in the future since it is necessary to keep compatibility.

It is okay to use the current way.